### ISSUE TITLE
Package no longer includes the build tools

### ISSUE THREAD

**Issue by khalwat** [akamai/boomerang#260]
I've been using a custom script to build Boomerang myself using your `grunt` build process.

I updated to the most recent version of Boomerang, only to find the build tools missing when installed via `npm`:

```bash
vagrant@homestead ~/webdev/craft/craft-webperf/node_modules/boomerangjs (develop) $ ls -al
total 676
drwxr-xr-x   13 vagrant vagrant    442 May 21 14:05 .
drwxr-xr-x 1099 vagrant vagrant  37366 May 21 13:58 ..
-rw-r--r--    1 vagrant vagrant 122532 May 21 13:55 boomerang.js
-rw-r--r--    1 vagrant vagrant   1428 May 21 13:55 bower.json
drwxr-xr-x   14 vagrant vagrant    476 May 21 13:56 doc
-rw-r--r--    1 vagrant vagrant   1695 May 21 13:55 LICENSE.txt
drwxr-xr-x  883 vagrant vagrant  30022 May 21 14:03 node_modules
-rw-r--r--    1 vagrant vagrant   5312 May 21 13:55 package.json
-rw-r--r--    1 vagrant vagrant 437248 May 21 14:03 package-lock.json
drwxr-xr-x   31 vagrant vagrant   1054 May 21 14:00 plugins
-rw-r--r--    1 vagrant vagrant    239 May 21 14:03 plugins.json
-rw-r--r--    1 vagrant vagrant    788 May 21 14:00 plugins.json.bak
-rw-r--r--    1 vagrant vagrant  18580 May 21 13:55 README.md
```

What I've done in the interim is `git clone`d the repo down, and doing my build that way. But I'm thinking these build tools should be part of the `npm install`?

**Comment by nicjansma** [comment]
That's fair, yeah.  Let me figure out what's required for the package to build after the NPM install.

**Comment by khalwat** [comment]
FWIW it used to all be there -- I guess this was part of a tidying up process?

**Comment by nicjansma** [comment]
Correct!

**Comment by nicjansma** [comment]
I think this should be fixed via #261

**Comment by khalwat** [comment]
Looks like that PR still hasn't been merged @nicjansma ?
